src/main/java/org/folio/dew/batch/bulkedit/jobs/BulkEditInstanceListProcessor.java

```diff
@@ -0,0 +1,33 @@
+package org.folio.dew.batch.bulkedit.jobs;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.log4j.Log4j2;
+import org.folio.dew.domain.dto.InstanceCollection;
+import org.folio.dew.domain.dto.InstanceFormat;
+import org.folio.dew.error.BulkEditException;
+import org.springframework.batch.core.configuration.annotation.StepScope;
+import org.springframework.batch.item.ItemProcessor;
+import org.springframework.stereotype.Component;
+
+import java.util.List;
+
+import static org.folio.dew.utils.Constants.NO_MATCH_FOUND_MESSAGE;
+
+@Component
+@StepScope
+@RequiredArgsConstructor
+@Log4j2
+public class BulkEditInstanceListProcessor implements ItemProcessor<InstanceCollection, List<InstanceFormat>> {
+  private final BulkEditInstanceProcessor bulkEditInstanceProcessor;
+
+  @Override
+  public List<InstanceFormat> process(InstanceCollection instances) {
+    if (instances.getInstances().isEmpty()) {
```

Comment:
Do we need to throw exception here and terminate process? From the first look it makes sense to complete processing with empty list in case instances list is empty (i.e. identifiers cannot be found). Could you please doublecheck?

Reply:
@khandramai 
The processor implementation pattern is taken from the BulkEditItemListProcessor

Such behavior doesn't terminate the process.
The exception is thrown for the case, the exact Instance identifier refers to nothing, and then BulkEditSkipListener puts the error message into error CSV, meanwhile, the flow continues processing other items.

such the case is verified/covered by BulkEditTest.bulkEditInstanceJobTestWithErrors - test